civihr_employee_portal/js/reports.js

```diff
@@ -416,6 +435,12 @@
     if (!this.jsonUrl) {
       return;
     }
+
+    //On load of page the form values are not appended to query string.
+    if(filterValues === '?' && that.reportName === 'leave_and_absence') {
```

Comment:
Not sure about JS coding standard but should there be a space after `if`?

Reply:
Looking at the other conditions in the file, looks like there should be a space.